File: scala/tests/fixtures/hello/HelloTest.scala
// M23 seed Scala test; consumer of scala_test (ScalaTest).
package hello

import org.scalatest.flatspec.AnyFlatSpec

class HelloTest extends AnyFlatSpec {
  "Hello.hello" should "greet by name" in {
    assert(Hello.hello("world") == "hello world")
  }
}

File: typescript/tests/fixtures/hello/hello_test.js
// Seed Jest test over the tsc-compiled output. Static-import coverage
// of TypeScript sources needs the transform slice (see
// //javascript/tests/fixtures/hello:hello_test); until then the test imports the compiled
// `./hello.js` from the runfiles tree via `data = [":hello_lib"]`.
import { hello } from "./hello.js";

test("greets by name", () => {
	expect(hello("world")).toBe("hello world");
});
